iceberg/iceberg-handler/src/test/results/positive/show_partitions_test.q.out

```diff
@@ -0,0 +1,214 @@
+POSTHOOK: query: select * from default.ice2.partitions order by `partition`
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+{"d_part":10,"e_part":5}	2	1	0
+{"d_part":2,"e_part":5}	1	1	0
+{"d_part":3,"e_part":4}	2	1	0
+PREHOOK: query: show partitions ice2
+PREHOOK: type: SHOWPARTITIONS
+PREHOOK: Input: default@ice2
+POSTHOOK: query: show partitions ice2
+POSTHOOK: type: SHOWPARTITIONS
+POSTHOOK: Input: default@ice2
+Spec-id=0*/d_part=10/e_part=5
+Spec-id=0*/d_part=2/e_part=5
+Spec-id=0*/d_part=3/e_part=4
+PREHOOK: query: ALTER TABLE ice2 SET PARTITION SPEC (c)
+PREHOOK: type: ALTERTABLE_SETPARTSPEC
+PREHOOK: Input: default@ice2
+POSTHOOK: query: ALTER TABLE ice2 SET PARTITION SPEC (c)
+POSTHOOK: type: ALTERTABLE_SETPARTSPEC
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: default@ice2
+PREHOOK: query: select * from default.ice2.partitions order by `partition`
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice2
+PREHOOK: Output: hdfs://### HDFS PATH ###
+POSTHOOK: query: select * from default.ice2.partitions order by `partition`
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+{"d_part":10,"e_part":5,"c":null}	2	1	0
+{"d_part":2,"e_part":5,"c":null}	1	1	0
+{"d_part":3,"e_part":4,"c":null}	2	1	0
+PREHOOK: query: show partitions ice2
+PREHOOK: type: SHOWPARTITIONS
+PREHOOK: Input: default@ice2
+POSTHOOK: query: show partitions ice2
+POSTHOOK: type: SHOWPARTITIONS
+POSTHOOK: Input: default@ice2
+Spec-id=0/d_part=10/e_part=5
+Spec-id=0/d_part=2/e_part=5
+Spec-id=0/d_part=3/e_part=4
+PREHOOK: query: insert into ice2 values  ('aa', 1, 2, 3, 4), ('aa', 1, 2, 3, 4), ('aa', 1, 3, 2, 5), ('aa', 1, 4, 10, 5), ('aa', 1, 5, 10, 5)
+PREHOOK: type: QUERY
+PREHOOK: Input: _dummy_database@_dummy_table
+PREHOOK: Output: default@ice2
+POSTHOOK: query: insert into ice2 values  ('aa', 1, 2, 3, 4), ('aa', 1, 2, 3, 4), ('aa', 1, 3, 2, 5), ('aa', 1, 4, 10, 5), ('aa', 1, 5, 10, 5)
+POSTHOOK: type: QUERY
+POSTHOOK: Input: _dummy_database@_dummy_table
+POSTHOOK: Output: default@ice2
+PREHOOK: query: select * from default.ice2.partitions order by `partition`
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice2
+PREHOOK: Output: hdfs://### HDFS PATH ###
+POSTHOOK: query: select * from default.ice2.partitions order by `partition`
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+{"d_part":10,"e_part":5,"c":null}	2	1	0
+{"d_part":2,"e_part":5,"c":null}	1	1	0
+{"d_part":3,"e_part":4,"c":null}	2	1	0
+{"d_part":null,"e_part":null,"c":2}	2	1	1
+{"d_part":null,"e_part":null,"c":3}	1	1	1
+{"d_part":null,"e_part":null,"c":4}	1	1	1
+{"d_part":null,"e_part":null,"c":5}	1	1	1
+PREHOOK: query: show partitions ice2
+PREHOOK: type: SHOWPARTITIONS
+PREHOOK: Input: default@ice2
+POSTHOOK: query: show partitions ice2
+POSTHOOK: type: SHOWPARTITIONS
+POSTHOOK: Input: default@ice2
+Spec-id=0/d_part=10/e_part=5
```

Comment:
maybe
````
(spec-id=0)/d_part=10/e_part=5
(spec-id=0/d_part=2/e_part=5
(spec-id=0)/d_part=3/e_part=4
(spec-id=1)*/c=2
(spec-id=1)*/c=3
(spec-id=1)*/c=4
(spec-id=1)*/c=5  
````

Reply:
I think it will get confusing after some time when the number of partitions-evolution increase. I think it would be good to retain spec id with "*" to show the current spec id.

Reply:
Updated as discussed to :

```
{Spec-id=0}:d_part=10/e_part=5
{Spec-id=0}:d_part=2/e_part=5
{Spec-id=0}:d_part=3/e_part=4
{Spec-id=1}*:c=2
{Spec-id=1}*:c=3
{Spec-id=1}*:c=4
{Spec-id=1}*:c=5
```